doc/api/http.markdown

```diff
@@ -825,6 +825,20 @@ be called before [`response.end()`][] is called.
 If you call [`response.write()`][] or [`response.end()`][] before calling this, the
 implicit/mutable headers will be calculated and call this function for you.
 
+When headers have been set with [`response.setHeader()`][], they will be merged with
+any headers passed to [`response.writeHead()`][], with the headers passed to 
+[`response.writeHead()`][] given precedence.
+
+```js
+// returns content-type = text/plain
+const server = http.createServer( (req,res) => {
```

Comment:
I looked over other code examples in the same file to maintain consistency and found:

```
[a0viedo@arya node]$ grep doc/api/http.markdown -r -e "((" | wc -l
0
[a0viedo@arya node]$ grep doc/api/http.markdown -r -e "( (" | wc -l
3
```

I don't have any context in which one is following the style, though.